containers/home.tsx

```diff
@@ -1,66 +1,92 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import ListCardVideo from '../components/listCardVideo/listCardVideo';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
+}
+
+const Home: React.FC<iVideo> = () => {
+  const [viewNextVideo, setViewNextVideo] = useState(0);
+  const [showVideos, setShowVideos] = useState([]);
+
+  const cards = [
+    { title: 'Course 1', alt: 'img1', src: '1' },
+    { title: 'Course 2', alt: 'img2', src: '2' },
+    { title: 'Course 3', alt: 'img3', src: '3' },
+    { title: 'Course 4', alt: 'img4', src: '4' },
+    { title: 'Course 5', alt: 'img5', src: '5' },
+    { title: 'Course 6', alt: 'img6', src: '6' },
+  ];
+
+  useEffect(() => {
+    const videosToShow = [];
+
+    for (let i = viewNextVideo; i <= viewNextVideo + 3; i++) {
+      videosToShow.push(cards[i]);
+    }
+
+    setShowVideos(videosToShow);
+  }, [viewNextVideo]);
+
+  const arrowNext = () => {
+    if (viewNextVideo < 2) {
```

Comment:
Same comment about making conditions more clear. 😛

Reply:
Yes, this is so that it doesn't break in the click for the last video, since we sum the index for each iteration, in the console I checked and it comes up to the index 2 then it must be less than it and in this index sum the rest of index

Reply:
Another important comment, do you think that checking for a value lower than 2 will stand even if the max videos to show increases to 8, for example? Maybe we can find another condition to avoid this magic number. 😄

Reply:
Yes, I have tried some conditions dilemma mainly is in the cycle for, the condition that is max 3 it's equal = index max 0, 1, until index 2, I will try to add condition for more videos

Reply:
Thank you so much, @jonathanperazawl and @ivanovishado for your help and suggestion c: